initrd/etc/luks-functions

```diff
@@ -267,24 +268,37 @@ prepare_thumb_drive()
 
 select_luks_container()
 {
+	#TODO: extend logic to prompt for block devices with model if multiple LUKS are found on block device instead of partitions
+	# Then feed luks with those partitions so that reencrypt and passphrase change can use passphrase to test all selected
 	TRACE_FUNC
 	if [ -s /boot/kexec_key_devices.txt ]; then
 		DEBUG "Reusing known good LUKS container device from /boot/kexec_key_devices.txt"
-		DEBUG "LUKS container device: $(cut -d ' ' -f1 /boot/kexec_key_devices.txt)"
-		LUKS=$(cut -d ' ' -f1 /boot/kexec_key_devices.txt)
-	else
+		LUKS=$(cut -d ' ' -f1 /boot/kexec_key_devices.txt) 
+		DEBUG "LUKS container device: $(echo $LUKS)"
```

Comment:
```suggestion
		DEBUG "LUKS container device: $LUKS"
```

Remove useless echo

Reply:
> DEBUG "LUKS container device: $LUKS"

Would output on multiple lines in case of multiple LUKS containers. This is why its like it is @JonathonHall-Purism

Reply:
`DEBUG "LUKS container device: $LUKS"`

Would output multilines $LUKS content, where $LUKS in code is wrapped for multiline parsing and works as it is @JonathonHall-Purism

Reply:
@JonathonHall-Purism same reasoning of why https://github.com/linuxboot/heads/pull/1541/files#diff-2bca82e612385576acc67c4a36b2a9cf4d563281f5aa115b2b58fabf68e789e8R319 is there to convert with simplcity.

Reply:
And here https://github.com/linuxboot/heads/pull/1541/files#diff-2bca82e612385576acc67c4a36b2a9cf4d563281f5aa115b2b58fabf68e789e8R398

Reply:
OK, I see.  Those specific-diff links never work for me in Firefox, but I get the idea that the word split followed by echo is effectively combining the lines from the variable with space delimiters.